### ISSUE TITLE
Add support to reset command

### ISSUE THREAD

**Issue by fadc80** [DonJayamanne/gitHistoryVSCode#380]
What about to add the reset command as an option when you click on a specific commit? I mean as EGit Eclipse plugin does...

**Comment by DonJayamanne** [comment]
Do you mean revert a commit?

**Comment by fadc80** [comment]
I mean support to [git reset command](https://git-scm.com/docs/git-reset). For exemple, it can be used to rewind your current HEAD branch to the a specific revision:

```
git reset --soft HEAD~1
```

This command [undo the last local commit](https://www.git-tower.com/learn/git/faq/undo-last-commit).
It rewrites the history tree like a rebase does, but it is safe for just local commited changes.

The [EGit Eclipse plugin]() in his history view support this command.

**Comment by demurgos** [comment]
I agree, it would be nice to have a way to use `git reset --hard` or `git reset --soft` to move the current branch to a given commit. It is especially useful when working on a branch locally and then preparing it to be pushed by cleaning up the history.
